<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <!-- <param name="camera_info_url" value="package://usb_cam/camera_info/usb_cam.yaml"/> -->
    <param name="camera_info_url" value="package://usb_cam/camera_info/usb_cam.yaml"/>
    <param name="image_width" value="800" />
    <param name="image_height" value="600" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="base_camera_link_usb_cam" />
    <param name="io_method" value="mmap"/>
    <!--帧率-->
    <param name="framerate" value="15"/>
    <!--对比度-->
    <param name="contrast" value="29"/>
    <param name="gain" value="93"/>
    <!--最高亮度-->
    <param name="brightness" value="-33"/>
    <!--锐化-->
    <param name="sharpness" value="21"/>
    <!--自动对焦-->
    <param name="autofocus" value="false"/>
  </node>
</launch>

